Clinical-Genomics/schug
# Add Mane columns to transcripts file downloaded from Ensembl

### This PR adds | fixes:
- Fix #40 

### How to test:
- Download transcripts in build 38 and make sure that the Mane Transcripts columns contain data
- Downloading transcripts in build 38 should work as before

### Review:

This [version](https://semver.org/) is a:
